stories/SingleDatePicker_input.js

```diff
@@ -41,6 +54,18 @@ storiesOf('SDP - Input Props', module)
       displayFormat="MMM D"
     />
   ))
+  .addWithInfo('with show calendar icon', () => (
+    <SingleDatePickerWrapper
+      initialDate={moment().add(3, 'days')}
+      showDefaultInputIcon
+    />
+  ))
+  .addWithInfo('with custom show calendar icon', () => (
```

Comment:
Does this work right now? We don't appear to be passing through the `customInputIcon`